core/assets/bundles/bundle_de.properties

```diff
@@ -965,7 +965,7 @@ ability.statusfield = {0} Statusfeld
 ability.unitspawn = {0} Fabrik
 ability.shieldregenfield = Schild-regenerations-Feld
```

Comment:
Might as well change this to "Schildregenerationsfeld" while we're here. Or at least capitalise the middle word.